### ISSUE TITLE
Hive Upgrades part of the block but not all

### ISSUE THREAD

**Issue by Jynx2700** [NillerMedDild/Enigmatica6#2374]
Enigmatica 6 1.16.5
Resourceful Bees 0.6.7.2b

Also going to submit on Resourceful Bees as this could and possibly is on their end.

**Describe the bug**
Upgrading Hives with the actual upgrade only upgrades the block data to the new hive but the hive itself stays at the current previous tier. IE: T1 hive, upgrade it 3 times to T4, it has T4 stats but for crafting to an Apiary it thinks its a T1 still.

**Screenshots**
![Hive Upgrades](https://user-images.githubusercontent.com/12675459/121058667-37c2f200-c786-11eb-85ec-07a09baffee8.png)

**Comment by TemperedFool** [comment]
Yeah, this is the way this works. I'm not sure what exactly they're planning to do with this, but they are currently working on a pretty major revamp and I think this is one of the things they're addressing. Not a lot we can do on our end I think, without just dumbing down the recipes so that the apiary requires any hive.

**Comment by Jynx2700** [comment]
I kinda figured it was on their end, I already submitted a report for them specifically... a quick fix that could be done would be to just simply block the crafting of them... Seems like its their only use and this would at least prevent people from stumbling into the bug like I did... Appreciate the work

**Comment by TemperedFool** [comment]
Per Epic it looks like hives as we know them are going away: https://github.com/Resourceful-Bees/ResourcefulBees/issues/258#issuecomment-856109343

I think if/when the current update releases, we're just going to have to review the changes and see how to integrate them.
